src/mann_whitney.jl

```diff
@@ -227,7 +228,8 @@ function pvalue(x::ApproximateMannWhitneyUTest; tail=:both)
         1
     else
         if tail == :both
-            2 * ccdf(Normal(), abs(x.mu - 0.5 * sign(x.mu))/x.sigma)
+            p = ccdf(Normal(), abs(x.mu - 0.5 * sign(x.mu))/x.sigma)
+            min(2 * p, 1.0)
```

Comment:
A comment related to this function (and your comment that a `Float64` will be returned) but unrelated to this change: this function is not type stable since it returns `1` in the other branch above.